core/invoker/src/main/resources/application.conf

```diff
@@ -60,9 +60,13 @@ whisk {
     user-memory: 1024 m
     concurrent-peek-factor: 0.5 #factor used to limit message peeking: 0 < factor <= 1.0 - larger number improves concurrent processing, but increases risk of message loss during invoker crash
     akka-client:  false # if true, use PoolingContainerClient for HTTP from invoker to action container (otherwise use ApacheBlockingContainerClient)
-    prewarm-expiration-check-interval: 1 minute # period to check for prewarm expiration
+    prewarm-expiration-check-init-delay: 10 minute # the init delay time for the first check
+    prewarm-expiration-check-interval: 10 minute # period to check for prewarm expiration
     prewarm-expiration-check-interval-variance: 10 seconds # varies expiration across invokers to avoid many concurrent expirations
     prewarm-expiration-limit: 100 # number of prewarms to expire in one expiration cycle (remaining expired will be considered for expiration in next cycle)
+    prewarm-max-retry-limit: 5 # max retry limit for create prewarm
```

Comment:
Basically, this max limit is reached when the subsequent 5 retries are failed.
How about changing this to `max subsequent retry limit to create prewarm containers`?

Would be worth mentioning that the count would be reinitialized when creation is succeeded at any time.

Reply:
Already changed the comment to `max subsequent retry limit to create prewarm containers`